scss/_base_hr.scss

```diff
@@ -20,7 +20,31 @@
 
     // stylelint-disable-next-line selector-no-qualifying-type
     &.is-fixed-width {
-      @extend %fixed-width-container;
+      margin-left: auto;
+      margin-right: auto;
+
+      @media (max-width: $threshold-4-6-col) {
+        max-width: #{$grid-max-width - 2 * map-get($grid-margin-widths, small)};
+        width: calc(100% - #{2 * map-get($grid-margin-widths, small)});
+      }
+
+      @media (min-width: $threshold-4-6-col) and (max-width: $threshold-6-12-col) {
+        max-width: #{$grid-max-width - 2 * map-get($grid-margin-widths, medium)};
+        width: calc(100% - #{2 * map-get($grid-margin-widths, medium)});
+      }
+
+      @media (min-width: $threshold-6-12-col) {
+        max-width: #{$grid-max-width - 2 * map-get($grid-margin-widths, large)};
+        width: calc(100% - #{2 * map-get($grid-margin-widths, large)});
+      }
+
+      @at-root .row & {
```

Comment:
I don't think we are using `@at-root` anywhere at the moment, and I'm not sure why it would be needed here.

I think simply `.row &` does exactly the same.
And because `.u-fixed-width &` below is also the same these could be joined into :

```
.row &,
.u-fixed-width & { 
  width: 100%;
}
```

Reply:
![image](https://user-images.githubusercontent.com/2741678/109509718-1b356080-7a99-11eb-852b-74fdcf6a3425.png)

yes, that works, bu now I have a problem: autoformatter forces the two classes on separate lines, and the no-qualifying-selector directive can't be placed afer the comma. S I've broken the two up into individual selectors.

Reply:
fixed